aspnetcore/performance/caching/memory/sample/RPcache/Services/MyMemoryCache.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Microsoft.Extensions.Caching.Memory;
+
+namespace RPcache.Services
+{
+    #region snippet
+    // using Microsoft.Extensions.Caching.Memory;
+    public class MyMemoryCache : IMyMemoryCache
```

Comment:
Hmm the reason we had to wrap the MemoryCache in this way is because tag helpers rely on a size-less memory cache from DI, I'm not sure if that's a good idea. I'll check with the folks over MVC.